components/buttons.css

```diff
@@ -0,0 +1,177 @@
+@import url(/main.css);
+@import url(/components/avatar.css);
+
+.buttons-page {
+  padding: 0.5rem;
+  margin: 0.5rem;
+  background-color: white;
+  border-radius: 5px;
+}
+
+button:hover {
+  cursor: pointer;
+}
+
+h2,
+h4 {
+  padding: 15px 10px;
+}
+
+i {
+  color: inherit;
+  padding-right: 10px;
+}
+
+.btn-primary-simple,
+.btn-primary-outline,
+.btn-secondary-simple,
+.btn-secondary-outline,
+.btn-small,
+.btn-large,
+.btn-primary-icon,
+.btn-secondary-icon,
+.btn-disabled,
+.btn-disabled-2,
+.link-btn-primary,
+.link-btn-secondary {
```

Comment:
instead of using declaring for all the classes define a common class and use it wherever you need